src/ostorlab/agent/message/proto/v3/asset/file/api_schema/graphql/graphql.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto2";
+
+package ostorlab.agent.message.proto.v3.asset.file.api_schema.graphql;
+
+message Message{
+  optional bytes content = 1;
+  optional string url = 2;
+  optional string type = 3;
```

Comment:
what is type ?

Reply:
schema_type i'll change the name

Reply:
i meant why will it be used for ? from what i understood it can be graphql, wsdl etc right ? so no need since you have in proto name "graphql.proto"